platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -407,16 +408,28 @@ private void initializeGodot() {
 		GodotLib.io = io;
 		mSensorManager = (SensorManager)getSystemService(Context.SENSOR_SERVICE);
 		mAccelerometer = mSensorManager.getDefaultSensor(Sensor.TYPE_ACCELEROMETER);
-		mSensorManager.registerListener(this, mAccelerometer, SensorManager.SENSOR_DELAY_GAME);
 		mGravity = mSensorManager.getDefaultSensor(Sensor.TYPE_GRAVITY);
-		mSensorManager.registerListener(this, mGravity, SensorManager.SENSOR_DELAY_GAME);
 		mMagnetometer = mSensorManager.getDefaultSensor(Sensor.TYPE_MAGNETIC_FIELD);
-		mSensorManager.registerListener(this, mMagnetometer, SensorManager.SENSOR_DELAY_GAME);
 		mGyroscope = mSensorManager.getDefaultSensor(Sensor.TYPE_GYROSCOPE);
-		mSensorManager.registerListener(this, mGyroscope, SensorManager.SENSOR_DELAY_GAME);
 
 		GodotLib.initialize(this, io.needsReloadHooks(), getAssets(), use_apk_expansion);
 
+		String sensor_delay_string = GodotLib.getGlobal("input_devices/sensors/sensor_delay");
+		if (sensor_delay_string.equals("Fastest")) {
+			sensor_delay = SensorManager.SENSOR_DELAY_FASTEST;
+		} else if (sensor_delay_string.equals("Game")) {
+			sensor_delay = SensorManager.SENSOR_DELAY_GAME;
+		} else if (sensor_delay_string.equals("Normal")) {
+			sensor_delay = SensorManager.SENSOR_DELAY_NORMAL;
+		} else if (sensor_delay_string.equals("UI")) {
+			sensor_delay = SensorManager.SENSOR_DELAY_UI;
+		}
+
+		mSensorManager.registerListener(this, mAccelerometer, sensor_delay);
+		mSensorManager.registerListener(this, mGravity, sensor_delay);
+		mSensorManager.registerListener(this, mMagnetometer, sensor_delay);
+		mSensorManager.registerListener(this, mGyroscope, sensor_delay);
```

Comment:
This code doesn't look correct as the sensor manager is registering listeners here in 'onCreate', and it's also doing the same thing below in 'onResume'.
Based on the [Android activity lifecycle](https://developer.android.com/guide/components/activities/activity-lifecycle#alc), 'onResume' always follows 'onCreate' so the operation is just being repeated twice for no reasons.

I'd recommend deleting the section here in favor of registering the listener in onResume. This would also allow to resolve the crashing issue commented above, as by the time 'onResume' is invoked, GodotLib.setup would have already been invoked, preventing GodotLib.getGlobal from crashing.